### DISCUSSION TITLE
Order of .cake files under /content

### DISCUSSION THREAD

**Discussion by AakashM** [cake-build/cake#3325] [Q&A] [answered]
There are cake recipe nuget packages which contains .cake files under /content : https://cakebuild.net/docs/writing-builds/reusing-builds.
On loading the entire nuget package (like #load nuget:?package=MyRecipePackage), in which order are the .cake files under /content executed?

**Comment by devlead** [comment, marked as answer]
They're all merged and compiled with script containing `#load` directive. Essentially directive replaced with their contents.

If order crucial, then you can i.e. include/exclude with globber patterns.

**Reply by augustoproiete** [reply]
@AakashM remember you also have the option to declare utility functions and classes in the different `.cake` files inside the package, and then in the script that `#load`s the recipe you call a function that bootstraps the process in the order you want.
